src/newsletter-snackbar/newsletter-snackbar.js

```diff
@@ -1,12 +1,13 @@
 import React, { useState } from "react";
 import Snackbar from "@material-ui/core/Snackbar";
+import { subscribeNewsletter } from "../services/storage";
 
 const referredFromNewsletterSignup =
   document.referrer === "https://sendfox.com/";
 
+if (referredFromNewsletterSignup) subscribeNewsletter();
```

Comment:
I think it would be better to have this logic inside a `useLayoutEffect` hook rather than when the module loads.